sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java

```diff
@@ -51,17 +54,23 @@
 class DefaultWsClient implements WsClient {
 
   private final WsConnector wsConnector;
-  private final OrganizationService organizations;
-  private final PermissionsService permissionsService;
-  private final ComponentsService componentsService;
+  private final OrganizationService organizationsOld;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "OrganizationService"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)